src/Raven.Client/Http/Topology.cs

```diff
@@ -7,5 +7,10 @@ public sealed class Topology
         public long Etag;
         public List<ServerNode> Nodes;
         public List<ServerNode> Promotables = new List<ServerNode>();
+
+        public override string ToString()
+        {
+            return $"{{{nameof(Nodes)}: [{string.Join(",", Nodes)}], {nameof(Promotables)}: [{string.Join(",", Promotables)}], {nameof(Etag)}: {Etag}}}";
```

Comment:
I know it is debug.. but still, can `Nodes` be `null` here?

Reply:
Fixed